### PR TITLE
Add polls 2023-09-04

### PR DESCRIPTION
**Pull request by VoteWizard**
@Le-Bateleur-2023 @MakerJanSky
Please review and let me know if there are any concerns or areas for improvement.

### CODE REVIEW

File: governance/polls/Adjust Spark Protocol DAI Interest Rate Strategy Borrow Spread - September 4, 2023.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following change:
+* Adjust Spark Protocol DAI Interest Rate Strategy to increase the DAI Borrow Spread by 0.5% from 0% to **0.5%**.
```
Comment (MakerJanSky) [inline governance/polls/Adjust Spark Protocol DAI Interest Rate Strategy Borrow Spread - September 4, 2023.md:30]: The manual should be linked here:
https://manual.makerdao.com/module-index/module-dai-direct-deposit#spread

File: governance/polls/Adjust Spark Protocol Flash Loan Fee - September 4, 2023.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following change:
+* Adjust Spark Protocol [Flash Loan Fee](https://docs.sparkprotocol.io/developers/guides/flash-loans#flash-loan-fee) from 0.09% to **0%**.
```
Comment (MakerJanSky) [inline governance/polls/Adjust Spark Protocol Flash Loan Fee - September 4, 2023.md:30]: To match the rest of the format should read:
Adjust Spark Protocol to decrease the [Flash Loan Fee](https://docs.sparkprotocol.io/developers/guides/flash-loans#flash-loan-fee) by 0.09% from 0.09% to **0%**.

File: governance/polls/Adjust Spark Protocol Flash Loan Fee - September 4, 2023.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+The Flash Loan Fee was reset to 0.09% as a result of the last pool upgrade. The aim of this poll is to revert it back to its previous value.
+
+Please review the discussion [thread](https://forum.makerdao.com/t/upcoming-spell-proposed-changes/21801) and the [Spark Protocol documentation](https://docs.sparkprotocol.io/developers/guides/flash-loans) to help inform your position before voting.
```
Comment (MakerJanSky) [inline governance/polls/Adjust Spark Protocol Flash Loan Fee - September 4, 2023.md:34]: If you are going to link the Spark Protocol Documentation here, you should link it in the other two polls, too.

File: governance/polls/Adjust Spark Protocol Flash Loan Fee - September 4, 2023.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following change:
+* Adjust Spark Protocol [Flash Loan Fee](https://docs.sparkprotocol.io/developers/guides/flash-loans#flash-loan-fee) from 0.09% to **0%**.
+
+The Flash Loan Fee was reset to 0.09% as a result of the last pool upgrade. The aim of this poll is to revert it back to its previous value.
```
Comment (MakerJanSky) [inline governance/polls/Adjust Spark Protocol Flash Loan Fee - September 4, 2023.md:32]: Does this editorial serve a purpose? We do not include reasoning in the other posts.

File: governance/polls/JAT1 to JAT2 Asset Reallocation - September 4, 2023.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following JAT1 (James Asset Trust 1) to JAT2 (James Asset Trust 2) Asset Reallocation:
+* Selling the ETF products within JAT1.
+* Paying any proceeds from this sale after expenses in excess of 500 million USD into the surplus buffer.
```
Comment (MakerJanSky) [inline governance/polls/JAT1 to JAT2 Asset Reallocation - September 4, 2023.md:31]: Link to Surplus Buffer Documentation:

https://manual.makerdao.com/parameter-index/core/param-system-surplus-buffer

File: governance/polls/JAT1 to JAT2 Asset Reallocation - September 4, 2023.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+3. The Trustee will sign an agreement that effectively transfers the remaining USD 500M from the JAT1 cash assets into JAT2.
+4. Following the agreement, the Trustee will execute a transfer of the USD 500M from JAT1's cash account to JAT2's cash account.
+5. The Trustee will request that the newly transferred funds be added to the existing US Treasury 6 month ladder execution arrangement with Sygnum Bank.
+6. After all the funds have been moved from JAT1 to JAT2, the Trustee will proceed to formally dissolve JAT1 as it will no longer hold any assets.
+
+Please review the discussion [thread](https://forum.makerdao.com/t/proposal-to-revise-asset-allocation-of-jat1-and-jat2/21718) to inform your position on this issue before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
```
Comment (MakerJanSky) [inline governance/polls/JAT1 to JAT2 Asset Reallocation - September 4, 2023.md:46]: Does this change require an executive vote? Or will JAT1 and JAT2 be acting off the result of the poll?

**Review by MakerJanSky — CHANGES_REQUESTED** [review]
Kindly review the feedback for minor adjustments:

I suggest that we aim for more comprehensive descriptions in our poll summaries in the future. However, for this initial review, I didn't identify any specific actions to be taken.

File: governance/polls/Adjust Spark Protocol Flash Loan Fee - September 4, 2023.md (hunk 3 above)
Comment (VoteWizard) [reply governance/polls/Adjust Spark Protocol Flash Loan Fee - September 4, 2023.md:34]: The documentation is a fork of the Aave documentation and is not relevant to the other polls, but this poll changes a parameter that is included here.

File: governance/polls/Adjust Spark Protocol Flash Loan Fee - September 4, 2023.md (hunk 2 above)
Comment (VoteWizard) [reply governance/polls/Adjust Spark Protocol Flash Loan Fee - September 4, 2023.md:30]: Good catch!

File: governance/polls/Adjust Spark D3M Parameters - September 4, 2023.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-09-04T16:00:00
+end_date: 2023-09-07T16:00:00
+---
+# Poll: Adjust Spark Protocol D3M Parameters - September 4, 2023
+
+The Reserve Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the Stability Facilitators. The community can vote in this poll to express support or opposition to using the Spark Protocol Direct Deposit DAI Module (D3M) with the listed parameters. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, September 4 at 16:00 UTC.
```
Comment (patrick-j-govalpha) [inline governance/polls/Adjust Spark D3M Parameters - September 4, 2023.md:20]: Not a blocking suggestion, but DAI Direct Deposit matches the docs in the manual - https://manual.makerdao.com/module-index/module-dai-direct-deposit

```suggestion
The Reserve Governance Facilitators have placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the Stability Facilitators. The community can vote in this poll to express support or opposition to using the Spark Protocol DAI Direct Deposit Module (D3M) with the listed parameters. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, September 4 at 16:00 UTC.
```

File: governance/polls/Adjust Spark D3M Parameters - September 4, 2023.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to making the following parameter changes to the Spark Protocol DAI Direct Deposit Module (D3M):
```
Comment (patrick-j-govalpha) [inline governance/polls/Adjust Spark D3M Parameters - September 4, 2023.md:29]: see above comment - you've been inconsistent with the naming now, pick one :)

File: governance/polls/Adjust Spark D3M Parameters - September 4, 2023.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+* Increase the DIRECT-SPARK-DAI [Maximum Debt Ceiling](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) by 200 million DAI from 200 million DAI to **400 million DAI**.
```
Comment (patrick-j-govalpha) [inline governance/polls/Adjust Spark D3M Parameters - September 4, 2023.md:30]: Minor nitpick, but we would tend to include the technical variable name as well

so "Maximum Debt Ceiling (`line`)" - helps if people are reviewing exec code at any point to be familiar with var names.

Again, not blocking

**Review by MakerJanSky — APPROVED** [review]
Approved. In the future, I recommend including the documentation link for Spark on all Spark-related items. This is not a critical issue for the current situation.

**Review by patrick-j-govalpha — APPROVED** [review]
Looks fine. Approved. noted. Some non-blocking points

**Pull request merged**